src/tools/tool-options.h

```diff
@@ -126,7 +126,9 @@ struct ToolOptions : public Options {
       .add("--pass-arg",
            "-pa",
            "An argument passed along to optimization passes being run. Must be "
-           "in the form KEY@VALUE",
+           "in the form KEY@VALUE. If a pass --somepass is specified "
+           "multiple times, --pass-arg=somepass will apply to the closest "
+           "instance to the left. All other options apply to all instances.",
```

Comment:
This should also say "if KEY is the name of a pass, then [..]". Overall maybe:

> in the form KEY@VALUE. If KEY is the name of a pass then it applies to the closest instance of that pass before us. If KEY is not the name of a pass then it is a global option that applies to all pass instances that read it (later appearances of such global options override earlier ones).

Reply:
I like you suggestion better than mine and adopted it, although I removed the last part in parenthesis as the text already is quite long, especially when formatted into a column.

Reply:
Looks good!